xDripTests/Model/Glucose/GlucoseReadingTests.swift

```diff
@@ -476,4 +476,21 @@ final class GlucoseReadingTests: AbstractRealmTest {
         XCTAssert(reading.calculatedValue ~ 400.0)
         XCTAssertFalse(reading.hideSlope)
     }
+    
+    func testActiveSlope() {
```

Comment:
`< 0.0` and `> 0.0` is not enough here, let's do more precise tests like `~ 3.0`